code/modules/cargo/packs/organic.dm

```diff
@@ -47,51 +64,77 @@
 					/obj/item/storage/fancy/donut_box)
 	crate_name = "candy crate"
 
-/datum/supply_pack/organic/candy/randomised/fill(obj/structure/closet/crate/C)
-	var/list/L = contains.Copy()
-	for(var/i in 1 to num_contained)
-		var/item = pick_n_take(L)
-		new item(C)
+/datum/supply_pack/organic/fiestatortilla
+	name = "Fiesta Crate"
+	desc = "Spice up the kitchen with this fiesta themed food order! Contains 8 tortilla based food items, as well as a sombrero, moustache, and cloak!"
+	cost = 2750
+	contains = list(/obj/item/clothing/head/sombrero,
+					/obj/item/clothing/suit/hooded/cloak/david,
+					/obj/item/clothing/mask/fakemoustache,
+					/obj/item/reagent_containers/food/snacks/taco,
+					/obj/item/reagent_containers/food/snacks/taco,
+					/obj/item/reagent_containers/food/snacks/taco/plain,
+					/obj/item/reagent_containers/food/snacks/taco/plain,
+					/obj/item/reagent_containers/food/snacks/enchiladas,
+					/obj/item/reagent_containers/food/snacks/enchiladas,
+					/obj/item/reagent_containers/food/snacks/carneburrito,
+					/obj/item/reagent_containers/food/snacks/cheesyburrito,
+					/obj/item/reagent_containers/glass/bottle/capsaicin,
+					/obj/item/reagent_containers/glass/bottle/capsaicin)
+	crate_name = "fiesta crate"
 
-/datum/supply_pack/organic/randomized/chef
-	name = "Excellent Meat Crate"
-	desc = "The best cuts in the whole galaxy."
-	cost = 2000
-	contains = list(/obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/slime,
-					/obj/item/reagent_containers/food/snacks/meat/slab/killertomato,
-					/obj/item/reagent_containers/food/snacks/meat/slab/bear,
-					/obj/item/reagent_containers/food/snacks/meat/slab/xeno,
-					/obj/item/reagent_containers/food/snacks/meat/slab/spider,
-					/obj/item/reagent_containers/food/snacks/meat/rawbacon,
-					/obj/item/reagent_containers/food/snacks/spiderleg,
-					/obj/item/reagent_containers/food/snacks/carpmeat,
-					/obj/item/reagent_containers/food/snacks/meat/slab/human)
-	crate_name = "food crate"
+/datum/supply_pack/organic/pastry
```

Comment:
I mean, we already have the candies crate. Just merge the two together, you sweet tooth.

Reply:
This is meant for birthday parties and such as it gives you several cupcakes and plain cakes, as well as a reliable way to get donuts and sprinkles from a reliable source.

Reply:
Just merge the two together, you'll eventually get a couple cupcakes and plain cakes anyway.